packages/protocol/releaseData/nonstandard/baklavaUpdate.json

```diff
@@ -0,0 +1,7 @@
+{
+  "GrandaMento": ["0x000000000000000000000000000000000000ce10", "0x0000000000000000000000000000000000000000", "300000000000000000000000", "0", "1209600"],
```

Comment:
Should we have set some approver for testing purposes? And some spread?

Reply:
Added the approver, spread is set as zero in mainnet.